### PR TITLE
Add card and list

### PR DESCRIPTION
**Pull request by svenvonwerner**

### CODE REVIEW

File: src/App copy [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Card from './components/ChallengeCard.js';
+import styled from 'styled-components';
+
+function App() {
```
Comment (celinevo) [inline src/App copy:4]: Is there a reason, why you copied `App` and didn´t just use `App.js`, that is already there after installing react-app? If you continue using the copy you need to put `.js` behind the name (like `AppCopy.js`). :)

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+  * {
+    box-sizing: border-box;
+  }
+
+  
+  body {
+    margin:1rem;
```
Comment (celinevo) [inline src/GlobalStyles.js:10]: Here is a missing white-space ;)

File: src/GlobalStyles.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    font-family: 'Lato', sans-serif;
+    background-color:#EDEFF4;
```
Comment (celinevo) [inline src/GlobalStyles.js:12]: And here is one too!

File: src/GlobalStyles.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  }
+
+  h1 {
+   font-size: 1.5rem;
+   font-weight:normal;
```
Comment (celinevo) [inline src/GlobalStyles.js:17]: And here :D

File: src/components/ChallengeCardList.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+import Card from './ChallengeCard.js';
+import { nanoid } from 'nanoid';
+
+const cards = [
+  {
+    id: nanoid(),
+    headlineCard: 'This is a Challenge',
```
Comment (celinevo) [inline src/components/ChallengeCardList.js:8]: You use `headlineCard` and `descriptionCard`. I think you can just use `headline` and `description`. It is still easy to understand and the name is smaller.

**Review by celinevo — COMMENTED** [review]
I found a couple little things, but nothing mayor. The only thing you should reconsider is your `App copy` file. Otherwise it looks very good!

**Review by hennifant — APPROVED** [review]
looks good to me :-) keep it up.

**Comment by hennifant** [comment]
looks good to me :-) keep it up, I can only agree with Celine's comments.

File: src/GlobalStyles.js (hunk 2 above)
Comment (svenvonwerner) [reply src/GlobalStyles.js:10]: @celinevo "...missing white-space..." What do you mean?

File: src/GlobalStyles.js [hunk 6]
```diff
@@ -5,22 +5,24 @@ const GlobalStyle = createGlobalStyle`
...
-   font-size: 1.5rem;
-   font-weight:normal;
+    font-family: 'Lato', sans-serif;
+    font-size: 1.5rem;
+    font-weight: 300;
 
   }
   h2 {
     font-size: 1.0rem;
+    font-family: 'Lato', sans-serif;
```
Comment (MichiBerger) [inline src/GlobalStyles.js]: I think this "doppelt gemoppelt" ;) you´ve already set your font-family in the body...so you don´t have to declare it in your headings...just if you want to use another font-family...then you should declare it again!

**Pull request merged**